src/pages/workspace/reimburse/WorkspaceReimburseView.js

```diff
@@ -152,7 +152,9 @@ class WorkspaceReimburseView extends React.Component {
                     ]}
                 >
                     <View style={[styles.mv3, styles.flexRow, styles.flexWrap]}>
-                        <Text>
+                        {/* Workaround to fix https://github.com/Expensify/App/issues/17368.
+                            TODO: Remove `numberOfLines` once https://github.com/facebook/react-native/pull/35703 is merged and applied to our repo */}
```

Comment:
I would remove these comments, they clutter up the code and we don't know when that PR is going to be merged. 

Instead, I think it's better to link to https://github.com/facebook/react-native/pull/35703 in the PR `Details` section and provide the same context there.

Reply:
PR updated

Reply:
@s-alves10 thanks for updating the code. What I meant above is that you should add details from that comment to the PR `Details` section (see screenshot) so that anyone who comes back to this PR in the future knows the context of why this was modified like this and what PR we're waiting on.

<img width="610" alt="image" src="https://github.com/Expensify/App/assets/16747822/a49e285d-d163-4df2-a8b5-04d97986c399">